app/handlers.go

```diff
@@ -0,0 +1,66 @@
+package app
+
+import (
+	"GOBootcamp/app/models"
+	"encoding/csv"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+func (a *App) IndexHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "Welcome to POST API")
+	}
+}
+
+func (a *App) GetPostHandler() http.HandlerFunc {
```

Comment:
Same as above this should not be part of your app, but instead of a Handler which would contain a Service(s) as we are trying to create a clean architecture, the main principle is "keep every component at it's level" if there are doubts we could schedule a session to expand this :)